src/yaml/from_yaml.cr

```diff
@@ -89,6 +89,31 @@ def Array.new(ctx : YAML::ParseContext, node : YAML::Nodes::Node)
   end
 end
 
+def Set.new(ctx : YAML::ParseContext, node : YAML::Nodes::Node)
+  ctx.read_alias(node, self) do |obj|
+    return obj
+  end
+
+  ary = new
```

Comment:
Was this var copied from Array?
should it be something like `new_set` or something?